jujugui/static/gui/src/app/components/profile/bundle-list/bundle-list.js

```diff
@@ -195,6 +196,7 @@ ProfileBundleList.propTypes = {
   acl: shapeup.shape({
     isReadOnly: PropTypes.func.isRequired
   }).frozen.isRequired,
+  activeUsersProfile: PropTypes.bool.isRequired,
```

Comment:
isActiveUserProfile? Otherwise it seems a profile object or similar.

Reply:
done